extra/man/alacritty.5.scd

```diff
@@ -133,6 +133,12 @@ This section documents the *[window]* table of the configuration file.
 
 	Default: _1.0_
 
+*blur* <boolean> _(works on macOS/KDE Wayland)_
+
+	Request windowing system to blur content behind the window.
```

Comment:
```suggestion
	Request compositor to blur content behind transparent windows.
```

Window manager technically doesn't have the capability to do this, it's the compositor that can.